tests/foreman/api/test_contentmanagement.py

```diff
@@ -1225,6 +1226,26 @@ def test_positive_sync_container_repo_end_to_end(
             )
             assert result.status == 0
 
+        # Inspect the images with skopeo (BZ#2148813)
+        if not is_open('BZ:2148813'):
```

Comment:
@vsedmik As you mentioned this works locally for 6.12.z, so can this is_open block be removed?